src/mpr/__tests__/parseMPR.test.ts

```diff
@@ -96,16 +91,16 @@ describe('parseMPR', () => {
     for (const dat of dataFile.steps[0].data) {
       for (const key of Object.keys(dat.raw)) {
```

Comment:
Please use the 'for ... in' structure
`for (let key in {ab:1, cd:2}) {console.log(key)}`

Reply:
I personally use `const [key, value] = Object.entries()`

Reply:
Thanks both for checking it out and the suggestions. Neat destructuring.